parley/src/layout/line/greedy.rs

```diff
@@ -559,7 +561,7 @@ impl<'a, B: Brush> BreakLines<'a, B> {
             // Round block/vertical axis metrics
             line.metrics.ascent = line.metrics.ascent.round();
             line.metrics.descent = line.metrics.descent.round();
-            line.metrics.leading = (line.metrics.leading * 0.5).round() * 2.;
+            line.metrics.leading = line.metrics.line_height - (line.metrics.ascent + line.metrics.descent);
```

Comment:
`line.metrics.line_height` is not rounded at any point as it is, should be rounded here or when it is computed.

Reply:
I have gone with rounding here as this is where it is done for the other metrics